src/store/ChartState.js

```diff
@@ -23,6 +23,7 @@ class ChartState {
     @observable isOnPagination = false;
     @observable paginationEndEpoch;
     @observable isChartClosed = false;
+    @observable shouldMinimiseLastDigits = false;
```

Comment:
@nazaninreihani, while `shouldMinimiseLastDigits` is a requirement of `LastDigitStats` it would be better to move it to `LastDigitStatsStore` as a computed property.